libavcodec/vaapi_encode.c

```diff
@@ -2401,6 +2438,18 @@ av_cold int ff_vaapi_encode_init(AVCodecContext *avctx)
     if (err < 0)
         goto fail;
 
+    if (ctx->codec->block_size) {
+        ctx->codec->block_size(avctx);
```

Comment:
block_size is too narrow.
could we do this in ctx->codec->configure(avctx);?
or create an new function named as ctx->codec->get_encoder_caps.

Reply:
Yes, agree. will fix in next version.